index.html

```diff
@@ -35,6 +35,14 @@
                     refreshSidebar()
                 }
             })
+
+            RED.menu.addItem('menu-item-subflow', {
+                id: "menu-item-subflow-export",
+                label: 'Export Subflow',
+                onselect: function() {
+                    subMenuItem()
```

Comment:
I think this should be `showSidebar()`? 

I did say untested 😜 


```suggestion
                    showSidebar()
```